reporting-app/app/models/activity_report_application_form.rb

```diff
@@ -30,21 +30,61 @@ def activities_by_month
     @activities_by_month ||= activities.group_by(&:month)
   end
 
+  def certification
```

Comment:
there are bunch of design violations based on our standards : 
- Model queries other aggregates
- Service instantiated in model
- Business logic in model
- Code duplication